### ISSUE TITLE
stage editor

### ISSUE THREAD

**Issue by NimuDevALTACC** [ShadowMario/FNF-PsychEngine#10654]
### What feature do you want to get added on the **base** engine?

plz

### To test your sight, and reliability, please select the option of what should NOT be requested.

Proper credit that was forgotten

**Comment by MemeHoovy** [comment]
> ### What feature do you want to get added on the **base** engine?
> plz
> 
> ### To test your sight, and reliability, please select the option of what should NOT be requested.
> Proper credit that was forgotten

no, just no, you deserve no credit, you deserve no credit in life, you deserve nothing at all you worthless cocksucking bitch, you don't even deserve a github account, you need to fuck off with your shitty pull requests, you need to fuck off from github, you need to fuck off in general, **just fuck off** you wretched scum

**Comment by Mastercopper2346** [comment]
https://m.youtube.com/watch?v=ZOloKO_he5Y

Let this bear clean you sins

**Comment by thecosmicgamer29** [comment]
what 'credit'
